frontend/src/app/applications/application-submitted/application-submitted.component.ts

```diff
@@ -4,32 +4,27 @@ import { Component, OnInit } from '@angular/core';
 import { Router, ActivatedRoute, Params } from '@angular/router';
 
 @Component({
-  providers: [ ApplicationService ],
+  providers: [ApplicationService],
   selector: 'app-application-submitted',
   templateUrl: './application-submitted.component.html'
 })
 export class ApplicationSubmittedComponent implements OnInit {
-
   application = new SpecialUseApplication();
+  // TODO Use type in teh template to filter content. type = 'noncommercial' or 'temp-outfitter'
+  type: string;
 
-  constructor(
-    private applicationService: ApplicationService,
-    private route: ActivatedRoute,
-    private router: Router
-  ) { }
+  constructor(private applicationService: ApplicationService, private route: ActivatedRoute, private router: Router) {}
 
   ngOnInit() {
     this.route.params.subscribe(params => {
-      console.log(params['id']);
-      this.applicationService.getOne(params['id'])
-        .subscribe(
-          application => this.application = application,
-          (e: any) => {
-            // TODO: Get feedback from Peter on how to handle
-            console.log(e);
-          }
-        );
-      });
+      this.type = params['type'];
+      this.applicationService.getOne(params['id']).subscribe(
+        application => (this.application = application),
+        (e: any) => {
+          // TODO: Get feedback from Peter on how to handle
```

Comment:
Can we catch this and display an alert error message to the user when something goes wrong?